src/Calculator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Acme;
+
+class Calculator
+{
+    public $displayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaaayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy;
+
+    function div(int $a, int $b): int
+    {
+        return $a / $b;
+    }
+
+    var $aaa;
+
+    var $bbb;
+
+    public function sayHello(DateTimeImutable $date): void
+    {
+        echo 'Hello, ' . $date->format('j. n. Y');
+    }
+
+    public function aaa()
+    {
+        return in_array(1, [1, 2, 3]);
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Call to function in_array() requires parameter #3 to be set.